testgen/icu75/message_fmt2/formatter/integer.json

```diff
@@ -0,0 +1,55 @@
+{
+  "scenario": "Integer formatter",
+  "defaultTestProperties": {
+    "testSubtype": "formatter",
+    "locale": "en-US"
+  },
+  "tests": [
+    {
+      "pattern": "hello {4.2 :integer}",
+      "verify": "hello 4"
+    },
+    {
+      "pattern": "hello {-4.20 :integer}",
+      "verify": "hello -4"
+    },
+    {
+      "pattern": "hello {0.42e+1 :integer}",
+      "verify": "hello 4"
+    },
+    {
+      "pattern": ".match {$foo :integer} one {{one}} * {{other}}",
+      "args": [
+        {
+          "argType": "number",
+          "name": "foo",
+          "value": 1.2
+        }
+      ],
+      "verify": "one"
+    },
+    {
+      "pattern": ".match {$foo :integer} 1.2 {{=1.2}} one {{one}} * {{other}}",
+      "args": [
+        {
+          "argType": "number",
+          "name": "foo",
+          "value": 1.2
+        }
+      ],
+      "verify": "=1.2"
```

Comment:
I predict this test expected value is not correct and will need changing, once we get the test data hooked up to the executor. Plural selection happens on the "formatted" number (more precisely: the formatted-to-parts result, which comes after rounding is performed, notation changes applied, units are added, etc.), not the input number. I would expect the result to be `other`. 

By contrast, the previous test case's expected value seems fine, since `1.2 :integer` -> `1` (formatted) -> `ONE` plural category (selection on formatted value).